src/test/kotlin/io/github/gradlenexus/publishplugin/TaskOrchestrationTest.kt

```diff
@@ -93,44 +96,77 @@ class TaskOrchestrationTest {
     }
 
-    @Test
-    internal fun `simplified close and release task without repository name should be available but trigger nothing if no repositories are configured`() {
+    @ParameterizedTest(name = "{0}")
+    @ValueSource(
+        strings = [
+            NexusPublishPlugin.SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME,
+            NexusPublishPlugin.SIMPLIFIED_CLOSE_TASK_NAME,
+            NexusPublishPlugin.SIMPLIFIED_RELEASE_TASK_NAME
+        ]
+    )
+    internal fun `simplified task without repository name should be available but trigger nothing if no repositories are configured`(simplifiedTaskName: String) {
         initSingleProjectWithDefaultConfiguration()
         project.extensions.configure<NexusPublishExtension> {
             repositories.clear()
         }
 
-        val simplifiedCloseAndReleaseTasks = project.getTasksByName(NexusPublishPlugin.SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME, true)
+        val simplifiedTasks = project.getTasksByName(simplifiedTaskName, true)
 
-        assertThat(simplifiedCloseAndReleaseTasks).hasSize(1)
-        assertThat(simplifiedCloseAndReleaseTasks.first().dependsOn).isEmpty()
+        assertThat(simplifiedTasks).hasSize(1)
+        assertThat(simplifiedTasks.first().dependsOn).isEmpty()
     }
 
-    @Test
-    internal fun `simplified closeAndRelease task without repository name should depend on all closeAndRelease (created one per defined repository)`() {
+    @ParameterizedTest(name = "{0}")
+    @MethodSource("simplified task without repository name should depend on all normal tasks (created one per defined repository) arguments")
+    internal fun `simplified task without repository name should depend on all normal tasks (created one per defined repository)`(simplifiedTaskName: String, sonatypeTaskName: String, otherNexusTaskName: String) {
         initSingleProjectWithDefaultConfiguration()
         project.extensions.configure<NexusPublishExtension> {
             repositories.create("otherNexus")
         }
 
-        val closeAndReleaseTask = getJustOneTaskByNameOrFail(NexusPublishPlugin.SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME)
+        val simplifiedTasks = getJustOneTaskByNameOrFail(simplifiedTaskName)
 
-        assertThat(closeAndReleaseTask.taskDependencies.getDependencies(null).map { it.name })
+        assertThat(simplifiedTasks.taskDependencies.getDependencies(null).map { it.name })
             .hasSize(2)
-            .contains("closeAndReleaseSonatypeStagingRepository")
-            .contains("closeAndReleaseOtherNexusStagingRepository")
+            .contains(sonatypeTaskName)
+            .contains(otherNexusTaskName)
     }
 
-    @Test
-    internal fun `description of simplified closeAndRelease task contains names of all defined Nexus instances`() {
+    private fun `simplified task without repository name should depend on all normal tasks (created one per defined repository) arguments`(): Stream<Arguments> = Stream.of(
```

Comment:
I would go for CsvSource, this seems a ton of code for what it does.

Reply:
fixed